src/main/webapp/well-known/assetlinks.json

```diff
@@ -1,15 +1,23 @@
 [{
-  "relation": ["delegate_permission/common.handle_all_urls"],
+  "relation": ["delegate_permission/common.get_login_creds"],
```

Comment:
Please don't change the relation right now. If it needs to be changed, I'll have to coordinate this with a few other changes. Here and below